oss_scripts/configure.sh

```diff
@@ -37,11 +37,17 @@ function is_macos() {
 # Remove .bazelrc if it already exist
 [ -e .bazelrc ] && rm .bazelrc
 
-if [[ $(pip show tensorflow) == *tensorflow* ]] || [[ $(pip show tf-nightly) == *tf-nightly* ]] ; then
+if [[ $(pip show tensorflow) == *tensorflow* ]] ||
+   [[ $(pip show tensorflow-macos) == *tensorflow-macos* ]] ||
+   [[ $(pip show tf-nightly) == *tf-nightly* ]]; then
   echo 'Using installed tensorflow.'
 else
   echo 'Installing tensorflow.'
-  pip install tensorflow==2.7.0rc0
+  if is_macos; then
```

Comment:
I believe you need the architecture check here too so only the M1s get the other lib

Reply:
I think it is better to choose tensorflow-macos here too.